python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/mpls.schema.yml

```diff
@@ -33,3 +33,255 @@ keys:
           ttl_exceeded_tunneling:
             type: bool
             description: Enables the MPLS tunneling of TTL exceeded ICMP replies.
+      rsvp:
+        type: dict
+        keys:
+          refresh:
+            type: dict
+            keys:
+              interval:
+                description: Time between refreshes.
+                type: int
+                convert_types:
+                  - str
+                min: 1
+                max: 65535
+              method:
+                description: |-
+                  Neighbor refresh mechanism.
+                  bundled: Refresh states using message identifier lists.
+                  explicit: Send each message individually.
+                type: str
+                valid_values:
+                  - bundled
+                  - explicit
+          authentication:
+            description: Cryptographic authentication.
+            type: dict
+            keys:
+              password_indexes:
+                type: list
+                primary_key: index
+                items:
+                  type: dict
+                  keys:
+                    index:
+                      description: Password index.
+                      type: int
+                      convert_types:
+                        - str
+                      min: 1
+                      max: 4294967295
+                    password_type:
+                      description: Authentication password type.
+                      type: str
+                      convert_types:
```

Comment:
Can we add default password_type: 7 similar to passwords in `monitor_telemetry_influx` schema?